test-four/main.css

```diff
@@ -3,7 +3,34 @@ body {
 }
 
 .container {
-  padding: 30px;
+  padding: 2.1vw;
+  margin: auto;
+  max-width: 1200px;
 }
 
-/* Place your CSS here... */
+.card-container {
+  display: grid;
```

Comment:
I like that your using CSS Grid, but in plain CSS I don't think this will not in IE?

Reply:
Yes in hindsight I should have used vendor prefixes to make my solution work in IE seeing as IE does have an implicit grid.